src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.Linux.cs

```diff
@@ -226,6 +226,7 @@ protected internal override void SetPinMode(int pinNumber, PinMode mode)
         /// </summary>
         /// <param name="pinNumber">The pin number in the driver's logical numbering scheme.</param>
         /// <param name="mode">The mode of a pin to set the resistor pull up/down mode.</param>
+        [MethodImpl(MethodImplOptions.NoOptimization)]
```

Comment:
Can you put a comment on top of this option to mention why do we require noOptimization here?

Reply:
I've put it inside since the comment looked weird under the docs